app/models/voxelytics/VoxelyticsService.scala

```diff
@@ -0,0 +1,205 @@
+package models.voxelytics
+
+import com.scalableminds.util.tools.{Fox, FoxImplicits}
+import models.user.User
+import models.voxelytics.VoxelyticsRunState.VoxelyticsRunState
+import play.api.libs.json.{JsObject, Json, OFormat}
+import utils.ObjectId
+
+import java.time.Instant
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+import scala.util.Try
+
+case class RunEntry(id: ObjectId,
+                    name: String,
+                    username: String,
+                    hostname: String,
+                    voxelyticsVersion: String,
+                    workflow_hash: String,
+                    workflow_yamlContent: String,
+                    workflow_config: JsObject,
```

Comment:
I guess the mix of snake case and camel case stems from what we get from voxelytics? Then that’s fine